packages/token-bridge-sdk/src/deposits/fetchDepositsFromSubgraph.ts

```diff
@@ -57,44 +54,28 @@ export const fetchDepositsFromSubgraph = async ({
     return []
   }
 
-  const res = await getL1SubgraphClient(l2ChainId).query({
-    query: gql`{
-        deposits(
-          where: {
-            sender: "${address}",
-            blockCreatedAt_gte: ${fromBlock},
-            blockCreatedAt_lte: ${toBlock}
-            ${searchString ? `transactionHash_contains: "${searchString}"` : ''}
-          }
-          orderBy: blockCreatedAt
-          orderDirection: desc
-          first: ${pageSize},
-          skip: ${pageNumber * pageSize}
-        ) {
-          receiver
-          sender
-          sequenceNumber
-          timestamp
-          tokenAmount
-          transactionHash
-          type
-          isClassic
-          id
-          ethValue
-          blockCreatedAt
-          l1Token {
-            symbol
-            decimals    
-            id
-            name
-            registeredAtBlock
-          }                  
-        }
-      }
-    `
+  // if dev environment, eg. tests, then prepend localhost
+  // Resolves: next-js-error-only-absolute-urls-are-supported in test:ci:sdk
+  const dev = process.env.NODE_ENV !== 'production'
+  const server = dev ? 'http://localhost:3000' : ''
+
+  const response = await fetch(`${server}/api/deposits`, {
+    method: 'POST',
```

Comment:
Let's try to follow RESTful architecture practices in our API. This should be a GET request with data passed in as query params: https://restfulapi.net/http-methods/#get

Reply:
Done